main.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	_ "github.com/udistrital/plan_adquisiciones_crud/routers"
+
+	"github.com/astaxie/beego"
+	"github.com/astaxie/beego/orm"
+	"github.com/astaxie/beego/plugins/cors"
+	_ "github.com/lib/pq"
+)
+
+func main() {
+	orm.RegisterDataBase("default", "postgres", beego.AppConfig.String("sqlconn"))
```

Comment:
en la linea 13 se be de hacer el ajuste para la conexión usando las variables de entorno, por ende el llamado a sqlconn de elimina y se debe realizar el cambio, para guiarse puede utilizar el siguiente archivo

https://github.com/udistrital/evaluacion_crud/blob/2f65885d78d5bd4d327fee6448f0af8d9ac745e5/main.go#L17